README.md

```diff
@@ -0,0 +1,65 @@
+# SURVEYOR
```

Comment:
@rafayet-monon We have `git-template` 🙌 
What do you think if we have the same format like them? 🤔
https://github.com/nimblehq/git-template/blob/master/README.md

What we need to add more is about the `License` and `About` section. 👍

Reply:
Updated requested changes!